test/NuGet.Clients.FuncTests/NuGet.MSSigning.Extensions.FuncTest/MSSignCommandTestFixture.cs

```diff
@@ -106,8 +107,8 @@ private async Task<CertificateAuthority> CreateDefaultTrustedCertificateAuthorit
             _trustedTimestampRoot = TrustedTestCert.Create(
                 rootCertificate,
                 StoreName.Root,
-                StoreLocation.LocalMachine);
-
+                (RuntimeEnvironmentHelper.IsWindows || RuntimeEnvironmentHelper.IsMacOSX) ? StoreLocation.LocalMachine : StoreLocation.CurrentUser);
```

Comment:
This is weird. I would expect Linux and OSX to have very similar locations for their data. What's this based on?

Reply:
Hi Kat, thanks for your question! 
This is from https://github.com/dotnet/runtime/blob/master/docs/design/features/cross-platform-cryptography.md#x509store
On Windows the X509Store class is a representation of the Windows Certificate Store APIs, and work the same as they did on .NET Framework.
On Linux the X509Store class is a projection of system trust decisions (read-only), user trust decisions (read-write)
On macOS the X509Store class is a projection of system trust decisions (read-only), user trust decisions (read-only)
So I use different approaches for Linux and Mac.
Do you think it's necessary to add some comments in our code to mention the background? Thanks!

Reply:
I think a comment with that link would be a great idea!

Reply:
Comments added. Thanks!